### PR TITLE
TRITON-2303 papi barfs on backend callback, also needs sapi update

### PR DESCRIPTION
**Pull request by danmcd**
Tested at first on my standalone-ISO-install test by hotpatching and getting the `papi` service running.  Would appreciate a sample CLI sdc-papi to create a package.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "sdc-papi",
   "description": "Packages API",
-  "version": "7.2.3",
-  "author": "Joyent (joyent.com)",
+  "version": "7.2.4",
```
Comment (danmcd) [inline package.json:4]: @bahamat I'm right in bumping this, no?

File: package.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+  "author": "MNX Cloud (mnx.io)",
```
Comment (danmcd) [inline package.json:5]: Also, figure it's about time to start here.

**Review by danmcd — COMMENTED** [review]
Thank you for the submodule update.

**Comment by danmcd** [comment]
Also tested by `sdc-papi /packages` with looks-clean-to-me output.

File: package.json (hunk 1 above)
Comment (bahamat) [reply package.json:4]: Yep

**Review by bahamat — APPROVED** [review]

**Pull request merged**